frontend/src/talos/resource/resource.pb.ts

```diff
@@ -0,0 +1,83 @@
+/*
+* This file is a generated Typescript file for GRPC Gateway, DO NOT MODIFY
+*/
+
+import * as CommonCommon from "../../common/common.pb"
+import * as fm from "../../fetch.pb"
+
+export enum EventType {
+  CREATED = "CREATED",
+  UPDATED = "UPDATED",
+  DESTROYED = "DESTROYED",
+}
+
+export type Resource = {
+  metadata?: Metadata
+  spec?: Spec
+}
+
+export type Metadata = {
+  namespace?: string
+  type?: string
+  id?: string
+  version?: string
+  owner?: string
+  phase?: string
+  finalizers?: string[]
+}
+
+export type Spec = {
+  yaml?: Uint8Array
+}
+
+export type GetRequest = {
+  namespace?: string
+  type?: string
+  id?: string
+}
+
+export type Get = {
+  metadata?: CommonCommon.Metadata
+  definition?: Resource
+  resource?: Resource
+}
+
+export type GetResponse = {
+  messages?: Get[]
+}
+
+export type ListRequest = {
+  namespace?: string
+  type?: string
+}
+
+export type ListResponse = {
+  metadata?: CommonCommon.Metadata
+  definition?: Resource
+  resource?: Resource
+}
+
+export type WatchRequest = {
+  namespace?: string
+  type?: string
+  id?: string
+}
+
+export type WatchResponse = {
+  metadata?: CommonCommon.Metadata
+  eventType?: EventType
+  definition?: Resource
+  resource?: Resource
+}
+
+export class ResourceService {
```

Comment:
That's the main thing we are going to use in Typescript code: generated gateway client.